src/vma/sock/sock-redirect.cpp

```diff
@@ -2107,6 +2107,12 @@ pid_t fork(void)
 
 		safe_mce_sys().get_env_params();
 		vlog_start("VMA", safe_mce_sys().log_level, safe_mce_sys().log_filename, safe_mce_sys().log_details, safe_mce_sys().log_colors);
+#ifdef DEFINED_RDMA_LIB_RESET
+		if (rdma_lib_reset()) {
```

Comment:
@ilansmith @rafiw good idea.
Please consider using src/vma/util/verbs_extra.h and .cpp (instead of creating a new file rdma_cm.h) where all similar definitions are gathered